docs/data-sources/cloud_workload_security_agent_rules.md

```diff
@@ -3,12 +3,12 @@
 page_title: "datadog_cloud_workload_security_agent_rules Data Source - terraform-provider-datadog"
 subcategory: ""
 description: |-
-  Use this data source to retrieve information about existing Cloud Workload Security Agent Rules for use in other resources.
+  Use this data source to retrieve information about existing Cloud Workload Security Agent Rules for use in other resources (this data source is going to be deprecated soon).
```

Comment:
Can we somehow refer users to the new Terraform resource?  (Or maybe that is already here, just not showing up in the diff.)